src/components/map/components/map/index.js

```diff
@@ -284,8 +321,9 @@ export class VlMap extends vlElement(HTMLElement) {
   __initializeCoordinateSystem() {
     proj4.defs(
       "EPSG:31370",
-      "+proj=lcc +lat_1=51.16666723333333 +lat_2=49.8333339 +lat_0=90 +lon_0=4.367486666666666 +x_0=150000.013 +y_0=5400088.438 +ellps=intl +towgs84=-106.869,52.2978,-103.724,0.3366,-0.457,1.8422,-1.2747 +units=m +no_defs"
+      "+proj=lcc +lat_1=51.16666723333333 +lat_2=49.8333339 +lat_0=90 +lon_0=4.367486666666666 +x_0=150000.013 +y_0=5400088.438 +ellps=intl +towgs84=106.869,-52.2978,103.724,-0.33657,0.456955,-1.84218,1 +units=m +no_defs"
```

Comment:
Waarom deze change?

Reply:
verandert dit functioneel iets?

Reply:
Deze PR komt origineel van @SBylemans. Om eerlijk te zijn weet ik dit ook niet atm. Dus we checken dat best even met hem af?

Reply:
Er is een verschil tussen https://epsg.io/31370 en https://spatialreference.org/ref/epsg/belge-1972-belgian-lambert-72/ blijkbaar. Nog een beetje verder gezocht en dit gevonden https://github.com/OSGeo/PROJ/issues/1580. Er zijn dus blijkbaar 2 mogelijke manieren om van epsg:31370 te transformeren naar wsg:84. Daarin zit het verschil, ik weet niet welke de meest accurate is, misschien staat dat in de issue maar zo in detail heb ik het niet gelezen.

Reply:
Als er geen duidelijke reden is om het aan te passen, zou ik niet weten waarom we het zouden aanpassen, want het werkt nu ...

Reply:
Dus we reverten deze dan?

Reply:
Ik weet niet wat de drijfveer van @SBylemans was om het te veranderen ...

Reply:
Weet het ook niet meer, ik revert het